scripts/microbe_stage/microbe_editor/microbe_editor.as

```diff
@@ -119,7 +118,6 @@ class MicrobeEditor{
         // Get the species organelles to be edited
         auto@ templateOrganelles = cast<array<SpeciesStoredOrganelleType@>>(
             playerSpecies.organelles);
-
```

Comment:
Please don't remove completely reasonable whitespace, I can tolerate a little bit of such edits from @Untrustedlife as he has done so much.